src/objects.rs

```diff
@@ -19,6 +23,8 @@ pub enum ObjectShape {
 }
 
 /// Raw data belonging to an object. Used internally and for tile collisions.
+/// 
+/// Also see the [TMX docs](https://doc.mapeditor.org/en/stable/reference/tmx-map-format/#tmx-object).
```

Comment:
We don't need `#[allow(missing_docs)]` here, even though the members are not documented?

Reply:
I added it because the members are all pretty self-explanatory from the TMX docs linked. (It's there because of the members, not the parent enum).

Reply:
I don't mind the link to the TMX docs. My comment was not really referring to that line, but rather wondering why we didn't need `#[allow(missing_docs)]` for this struct, even though it doesn't have docs for its members.

Reply:
But that attribute is present on the struct. What do you mean?

Reply:
This change adds that attribyte to the `ObjectShape` enum, but I do not see it on the `ObjectData` struct. Am I blind?

Reply:
Oh lol, now I see what you mean. I thought this comment was referring to ObjectShape. No, docs are missing, that's why this PR is a draft. I still haven't finished some parts.